src/ucs/signal/Makefile.am

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (C) 2022, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+lib_LTLIBRARIES = libucs_signal.la
```

Comment:
can we have some test for it (that preloading libucs_signal.so prevents signals override)?
can also test manually and post output, or add CI test in next PR

Reply:
> can we have some test for it (that preloading libucs_signal.so prevents signals override)?
> can also test manually and post output, or add CI test in next PR

tested manually - signal/sigaction is hooked
will create test for it for CI